webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionDetails.js

```diff
@@ -31,8 +31,10 @@ const ContentViewVersionDetails = ({ cvId }) => {
   const tableConfigs = getCVVersionTableConfigs({ cvId, versionId });
 
   useEffect(() => {
-    dispatch(getContentViewVersionDetails(versionId, cvId));
-  }, [dispatch, versionId, cvId]);
+    if (isEmpty(response) && status === STATUS.PENDING) {
```

Comment:
do make sure that this call isn't expected by any of the edit/remove flows on the filter edit/remove or the repositories page.